web/packages/teleport/src/DesktopSession/useDesktopSession.tsx

```diff
@@ -185,6 +186,15 @@ export default function useDesktopSession() {
     }
   };
 
+  const onCtrlAltDel = () => {
+    if (!tdpClient) {
+      return;
+    }
+    tdpClient.sendKeyboardInput('ControlLeft', ButtonState.DOWN);
+    tdpClient.sendKeyboardInput('AltLeft', ButtonState.DOWN);
+    tdpClient.sendKeyboardInput('Delete', ButtonState.DOWN);
```

Comment:
Sort of worried that not sending an "UP" could cause weird issues, but I haven't been able to induce anything.

Reply:
Agreed, I thought I was going to have to do that too, but this is the first thing I tried and I couldn't get it to break!